extensions/mssql/src/sqlClusterLookUp.ts

```diff
@@ -149,7 +149,7 @@ async function createSqlClusterConnInfo(sqlConnInfo: azdata.IConnectionProfile |
 
 async function getClusterController(controllerEndpoint: string, connInfo: ConnectionParam): Promise<bdc.IClusterController | undefined> {
 	const bdcApi = <bdc.IExtension>await vscode.extensions.getExtension(bdc.constants.extensionName).activate();
-	let authType: bdc.AuthType = connInfo.options[constants.authenticationTypePropName] === AuthType.Integrated ? 'integrated' : 'basic';
+	let authType: bdc.AuthType = connInfo.options[constants.authenticationTypePropName].toLowerCase() === AuthType.Integrated ? 'integrated' : 'basic';
```

Comment:
this should be a const, right?

Reply:
The 'integrated' and 'basic' values? Not needed, bdc.AuthType is `'integrated' | 'basic'` type

Reply:
I mean the authType variable

Reply:
Ah, yeah it could be